sql/log.cc

```diff
@@ -7761,7 +7761,7 @@ MYSQL_BIN_LOG::write_transaction_to_binlog(THD *thd,
   {
     DBUG_RETURN(0);
   }
-  else if (!(thd->variables.option_bits & OPTION_BIN_LOG))
+  else if (!WSREP(thd) && !(thd->variables.option_bits & OPTION_BIN_LOG))
```

Comment:
I'd prefer to isolate `!WSREP(thd) &&`  with 
```
#ifdef WITH_WSREP
 !WSREP(thd) &&
#endif
```
which would be also consistent with l.8671.

Reply:
Should be fixed now.